src/absorb.rs

```diff
@@ -103,28 +102,34 @@ pub trait AbsorbWithLength: Absorb {
 /// If `F1` and `F2` are the same type, return `input` but cast to `F2`, and panic otherwise.
 /// ## Panics
 /// This function will panic if `F1` is not equal to `F2`.
```

Comment:
```suggestion
/// If `F1` and `F2` have the same prime modulus, this method returns `Some(input)` but cast to `F2`, and returns `None` otherwise.
```

Reply:
fixed